src/market/market.py

```diff
@@ -163,15 +171,40 @@ def place_order(  # pylint: disable=too-many-arguments
 
         if payer == owner.public_key():
             raise ValueError("Invalid payer account")
+
+        should_wrap_sol = False
+        wrapped_sol_account = Account()
         if (side == side.Buy and self.state.quote_mint() == WRAPPED_SOL_MINT) or (
             side == side.Sell and self.state.base_mint == WRAPPED_SOL_MINT
         ):
-            # TODO: Handle wrapped sol account
-            raise NotImplementedError("WRAPPED_SOL_MINT is currently unsupported")
+            should_wrap_sol = True
```

Comment:
I don't think it will be easy to add a tests. We will need to create a SOL market and deposit the relevant accounts, which requires change in `crank`.

Reply:
I don't think this requires crank. We have all the code to create a SOL market and min_to relative accounts.

Reply:
If it's too much work can you add a TODO for testing?

Reply:
Added TODO